config/webpack.config.js

```diff
@@ -0,0 +1,27 @@
+import path from 'path';
+
+import module from './webpack.module';
+import plugins from './webpack.plugins';
+import resolve from './webpack.resolve';
+import devServer from './webpack.dev-server';
+
+export default (env, options) => {
+    const isProduction = options.mode === 'production';
+    return {
+        entry: {
+            app: './src/index.js',
+        },
+        output: {
+            path: path.resolve(__dirname, '../dist'),
+            publicPath: 'http://localhost:3000/',
```

Comment:
incorrect value - it should not depend on `http://localhost:3000/` - replace by `/`